src/index.test.ts

```diff
@@ -5,9 +5,9 @@ import { TsonOptions, TsonType, createTson } from "./index.js";
 import { expectError, waitError } from "./internals/testUtils.js";
 
 test("multiple handlers for primitive string found", () => {
-	const stringHandler: TsonType<string, string> = {
+	const stringHandler = {
 		primitive: "string",
-	};
+	} as TsonType<string, string>;
```

Comment:
This is equivalent to the other way around. Did you want to use the `satisfies` operator?

Reply:
Actually, this is necessary due to the changes in [`src/sync/syncTypes.ts`](https://github.com/trpc/tupleson/pull/87/files#diff-13a684546c72790218ed91e3314799e09dd271f95d5896b3145ac6ac6af60bfb)